collectors/ebpf.plugin/ebpf_filesystem.c

```diff
@@ -114,7 +114,8 @@ static void ebpf_create_fs_charts()
                               title,
                               EBPF_COMMON_DIMENSION_CALL, family,
                               NULL, NETDATA_EBPF_CHART_TYPE_STACKED, order, ebpf_create_global_dimension,
-                              filesystem_publish_aggregated, NETDATA_FILESYSTEM_MAX_BINS);
+                              filesystem_publish_aggregated,
+                NETDATA_EBPF_HIST_MAX_BINS);
```

Comment:
Please fix the formatting. There are 3 more similar places in this file).

Reply:
This was thanks `Clion` option to rename variable. :/ 
I fixed it, thank you!